activestorage/lib/active_storage/attached/macros.rb

```diff
@@ -83,7 +83,24 @@ def #{name}=(attachables)
         end
       CODE
 
-      has_many :"#{name}_attachments", -> { where(name: name) }, as: :record, class_name: "ActiveStorage::Attachment", inverse_of: :record
+      has_many :"#{name}_attachments", -> { where(name: name) }, as: :record, class_name: "ActiveStorage::Attachment", inverse_of: :record do
+        # Directly purges each associated attachment (i.e. destroys the blobs and
```

Comment:
The documentation will be lost now. We should document those methods in `Many` like the others using `:method:` and `:call-seq:` like:

https://github.com/rails/rails/blob/fa9e791e014a650f5ea6a14b283fed9621fc83e2/actionpack/lib/abstract_controller/callbacks.rb#L100